README.md

```diff
@@ -218,6 +218,144 @@ OpenStack | N | N | Y | N | N
 oVirt | N | N | Y | N | N
 VMWare | N | N | Y | N | N
 
+# Short lived Credentials with AWS Secure Token Service
+### Overview
+We can also make Openshift use temporary credentials for different components with AWS Secure Token Service (STS). It enables an authentication flow allowing a component to assume an IAM Role resulting in short-lived credentials. It also automates requesting and refreshing of credentials using OpenID Connect Identity Provider. Openshift can sign ServiceAccount token trusted by AWS IAM which can be projected into a Pod and used for authentication. The following is a diagram showing how it works.
+
+![sts flow](docs/sts_flow.png)
+
+### Changes in the Credentials Secret with STS
+
+Without STS we use access key id and secret access key
+
+```yaml
+aws_access_key_id = <access_key_id>
+secret_access_key = <secret_access_key>
+```
+
+With STS we use role and web identity token
+
+```yaml
+role_name = arn:...:role/some-role-name
+web_identity_token_file = /path/to/token
+```
+The token is short lived for an hour after which it is refreshed.
+
+### Steps to install an OpenShift Cluster with STS
+
+1. Use [this](https://quay.io/repository/jdiaz/custom-release) custom OpenShift release image
+2. Extract the AWS Credentials Request objects from the above release image. Please use [4.6](https://openshift-release-artifacts.apps.ci.l2s4.p1.openshiftapps.com/4.6.0/) openshift client (oc)
+   ```
+   oc adm release extract --cloud=aws --credentials-requests quay.io/jdiaz/custom-release:sts > credreqs.yaml
+   ```
```

Comment:
Maybe cover both cases:

2. Extract the AWS Credentials Request objects from the above release image.
With a 4.6 version of the `oc` CLI:
   ```
   oc adm release extract --cloud=aws --credentials-requests $RELEASE_IMAGE > credreqs.yaml
   ```
With a newser `oc` CLI it no longer prints to stdout:
   ```
   mkdir credreqs ; oc adm release extract --cloud=aws --credentials-request $RELEASE_IMAGE --to=./credreqs ; cat ./credreqs/*.yaml > credreqs.yaml
   ```

Reply:
maybe just let users get 4.7+ oc and document that., https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/latest-4.7/ ??

Reply:
The second example @joelddiaz puted has a wrong parameter , it should be  --credentials-requests, with "s", not --credentials-request.